pkg/controller/proposal/controller.go

```diff
@@ -456,7 +456,8 @@ func (r *Reconciler) reconcileCommit(ctx context.Context, proposal *configapi.Pr
 			if config.Values == nil {
 				config.Values = make(map[string]*configapi.PathValue)
 			}
-			for path, changeValue := range changeValues {
+			updChangeValues := doCascadingDelete(changeValues, config.Values)
```

Comment:
this `addDeletedChildren` should be conditional on a flag from modelPlugin ModelInfo

Reply:
Discussed in a call - no need to implement a flag so far